Samples/Samples.Android/Resources/xml/my_backup_rules.xml

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<full-backup-content>
+    <include domain="sharedpref" path="."/>
+    <exclude domain="sharedpref" path="${applicationId}.xamarinessentials.xml"/>
```

Comment:
If we merge this into the actual library, my concern is that we may conflict with another resource - especially since we have to specify this on the `<application>` element in the manifest. But... we can add this to the library and then let the user know that for better things they should use our resource - if they have their own, then they should add the line to theirs for better security.

Reply:
This is just for the sample app so I can test it. We wouldn't put it into the actual app as I don't think that is a good idea. 

We would just document it.